docker/ci/centos/Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM centos:7
 
+ENV TARGET_BIN=greptime
```

Comment:
Why modify this file? I think it's not a fundamental change for this fix. If you want to add TARGET_BIN, you can create another PR. Let’s keep the fix small and clean.

Reply:
I changed it back.